src/System.IO.Compression.Brotli/System/IO/Compression/Brotli.cs

```diff
@@ -37,9 +44,14 @@ private static TransformationStatus GetTransformationStatusFromBrotliDecoderResu
             return TransformationStatus.InvalidData;
         }
 
-        public static TransformationStatus Compress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, int quality = DefaultQuality, int windowSize = DefaultWindowSize, BrotliEncoderMode encMode = BrotliEncoderMode.Generic)
+        public static TransformationStatus Compress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, CompressionLevel quality = (CompressionLevel)DefaultQuality, int windowSize = DefaultWindowSize, BrotliEncoderMode encMode = BrotliEncoderMode.Generic)
```

Comment:
It seems a bit odd that we are casting the default value of the argument to CompressionLevel. I would imagine the default value would be part of the enum.